src/jit/morph.cpp

```diff
@@ -8730,6 +8716,21 @@ GenTree* Compiler::fgMorphCall(GenTreeCall* call)
             }
         }
 
+        void* pfnCopyArgs = nullptr;
+#if !defined(_TARGET_X86_)
```

Comment:
Seems like this will cause a noway assert later for x86?

Reply:
I had the same question after reviewing the changes.

Reply:
noway asserts are in blocks of code that are under amd64 and arm ifdefs. `pfnCopyArgs` is never used for x86.